local-container-endpoints/version/version.go

```diff
@@ -21,13 +21,13 @@ package version
 // you are doing.
 
 // Version is the version of ECS Local Container Endpoints
-const Version = "1.0.0"
+const Version = "1.0.1"
 
 // AgentVersionCompatibility is the Latest Agent Version that ECS Local Container Endpoints is compatible with
-const AgentVersionCompatibility = "1.26.0"
+const AgentVersionCompatibility = "1.27.0"
```

Comment:
does this mean that this release is NOT compatible with 1.26.0?

Reply:
@allisaurus @efekarakus Its the latest version of the Agent that we're compatible with. Since the Agent doesn't make backwards breaking changes with Task Metadata (or anything AFAIK), its implied that we're still compatible with everything < 1.27.0

Reply:
If it's an enhancement, should it be 1.26.1?